lib/rules/jsx-newline.js

```diff
@@ -46,6 +46,10 @@ module.exports = {
             default: false,
             type: 'boolean',
           },
+          checkStartEnd: {
+            default: false,
+            type: 'boolean',
+          },
```

Comment:
@ljharb this makes sense but I wonder if we shouldn't also include option for banning start/end newlines while forcing empty lines between children :thinking: 

So this might actually become an enum of `force` | `prevent` rather than boolean. What do you think, what should I go with?

Reply:
Or we might go with:

`undefined` and `false` = do nothing
`true` = behaves same as between children (as per `prevent`)
`force` = forces an empty line
`prevent` = prevents empty line

And it will still take `allowMultilines` into account